slider/position.py

```diff
@@ -16,3 +16,21 @@ class Position(namedtuple('Position', 'x y')):
     The visible region of the osu! standard playfield is [0, 512] by [0, 384].
     Positions may fall outside of this range for slider curve control points.
     """
+
+class Point(namedtuple('Point', 'x y offset')):
```

Comment:
Also, right now most things that have a `Position` also have a `time` or and `offset`. I wonder if we should consolidate all of these to use `Point` or whatever the new name is.